content/how-to/add-code-servers/bitbucket-cloud/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Adding a Bitbucket Cloud code server"
-description: "How to add a Bitbucket Cloud server to your tenant"
-lead: "How to add a Bitbucket Cloud server to your tenant"
```

Comment:
Actually, this was a good lead and description. They're different from the title and they accurately describe the page. I wouldn't remove these.